trade-finance-manager-api/src/v1/controllers/deal-cancellation/submit-deal-cancellation.controller.test.ts

```diff
@@ -1,18 +1,19 @@
 import { ObjectId } from 'mongodb';
 import httpMocks from 'node-mocks-http';
 import { HttpStatusCode } from 'axios';
-import { TestApiError, TfmDealCancellation } from '@ukef/dtfs2-common';
+import { AnyObject, TestApiError } from '@ukef/dtfs2-common';
+import { generateTfmAuditDetails } from '@ukef/dtfs2-common/change-stream';
 import { submitDealCancellation, SubmitDealCancellationRequest } from './submit-deal-cancellation.controller';
 
 const submitDealCancellationMock = jest.fn() as jest.Mock<Promise<void>>;
 
 jest.mock('../../services/deal-cancellation/deal-cancellation.service', () => ({
   DealCancellationService: {
-    submitDealCancellation: (dealId: string, dealCancellation: TfmDealCancellation) => submitDealCancellationMock(dealId, dealCancellation),
+    submitDealCancellation: (params: AnyObject) => submitDealCancellationMock(params),
   },
 }));
 
-const dealCancellation = {
+const cancellation = {
```

Comment:
I think `dealCancellation` is nicer - if we add other cancellation functionality that is _not_ for a deal this could cause confusion.

Equally we're in a `deal-cancellation` directory so maybe it's ok

Thoughts?

Reply:
No real preference - I just updated this so I could use `cancellation: dealCancellation` in the params. Have changed now